packages/aws-cdk/lib/cli/cdk-toolkit.ts

```diff
@@ -675,6 +676,48 @@ export class CdkToolkit {
     });
   }
 
+  /**
+   * Detect infrastructure drift for the given stack(s)
+   */
+  public async drift(options: DriftCommandOptions): Promise<number> {
```

Comment:
I think we'd want this code to live in `@aws-cdk/toolkit-lib`, and then call it from the CLI.

Right, @mrgrain ?

Reply:
Correct. See `refactor()` for how to do this correctly.